permissions/component-lib-access-modifier-annotation.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: "access-modifier"
+github: "jenkinsci/access-modifier"
+paths:
+- "org/kohsuke/access-modifier/annotation"
```

Comment:
No, `org/kohsuke/access-modifier-annotation` and similarly for the other components.

Reply:
@timja ^^^